tests/phpunit/wikiFunctionsTest.php

```diff
@@ -188,5 +188,33 @@ public function testCurlSetup() {
     curl_setup($ch, $url);
     $this->assertNull(NULL); // Just looking for code coverage and access of unset variables, etc.
   }
+
+  public function testPageIsRedirect() {
+    $page = new page();
+    $page->title = 'WP:UCB';
+    $this->assertEquals(1, $page->is_redirect()[0]); // Different code than non-class function is_redirect($text)
+  }
+  public function testPageTextFromTitle() { // Not a great test. Mostly just verifies no crashes in code
+    $page = new page();
+    $result = $page->get_text_from('User:Citation_bot');
+    $this->assertNotNull($result);
+  }
+  public function testEditSummary() {  // Not a great test. Mostly just verifies no crashes in code
+    $page = new Page();
+    $text = "{{Cite journal|pmid=9858586}}";
+    $page->parse_text($text);
+    $page->expand_text();
+    $this->assertNotNull($page->edit_summary());
+  }
+  
+  public function testMathInTitle() {
+    // This MML code comes from a real CroffRef search of DOI 10.1016/j.newast.2009.05.001
```

Comment:
Owch! (-:

Reply:
No kidding.  That is why I explcitly include the MML here.  I do not want CrossRef changing this on us.  We really need this test.